lib/cdn-site-hosting/cdn-site-hosting-construct.ts

```diff
@@ -3,8 +3,10 @@ import * as certificatemanager from "@aws-cdk/aws-certificatemanager";
 import * as cloudfront from "@aws-cdk/aws-cloudfront";
 import * as s3 from "@aws-cdk/aws-s3";
 import * as s3deploy from "@aws-cdk/aws-s3-deployment";
+import * as origins from "@aws-cdk/aws-cloudfront-origins";
 import { getSiteDomain } from "./utils";
 import { CommonCdnSiteHostingProps } from "./cdn-site-hosting-props";
+import { Duration } from "@aws-cdk/core";
```

Comment:
nitpick: I don't think we need this import, we should be able to access this from `cdk.Duration` imported on L1? 

I believe the preferred house style in this project is to import * as Y from X

Reply:
Fixed for you sir!